src/libostree/ostree-repo-private.h

```diff
@@ -150,7 +150,7 @@ struct OstreeRepo {
   uid_t owner_uid; /* Cache of repo's owner uid */
   uid_t target_owner_uid; /* Ensure files are chowned to this uid/gid */
   gid_t target_owner_gid;
-  guint min_free_space_percent; /* See the min-free-space-percent config option */
+  const char *min_free_space_str; /* See the min-free-space config option */
```

Comment:
I'm in favor of adding `min-free-space` as an alias for `min-free-space-percent`, but AFAICS this patch isn't doing that yet.